### ISSUE TITLE
External scripts still fail lint command

### ISSUE THREAD

**Issue by 43081j** [Polymer/polymer-cli#353]
0.14.0 fixed this issue with the build command (via a PR in polymer-build), but it seems the lint command has the same issue still.

See #261.

Any external scripts cause `polymer lint` to fail because there is no resolver specified for them. I guess this is because we just call polylint directly without specifying how to ignore such scripts.

From what I remember, though, polylint also currently has no way to pass such resolver config through, but I could be wrong.

**Comment by FredKSchott** [comment]
you're right, there isn't currently an option to define resolver behavior. See https://github.com/PolymerLabs/polylint/issues/141 for a relevant polylint (the tool our CLI uses internally) issue.

@justinfagnani is doing some work on our analyzer now, we should make sure we handle this when we update analyzer in polylint.
